rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -833,11 +833,24 @@ error_code cellGameGetSizeKB(vm::ptr<s32> size)
 		return CELL_GAME_ERROR_FAILURE;
 	}
 
+	if (Emu.GetCat() == "DG")
+	{
+		return CELL_GAME_ERROR_NOTSUPPORTED;
+	}
+
 	const std::string local_dir = !prm->temp.empty() ? prm->temp : vfs::get("/dev_hdd0/game/" + prm->dir);
 
 	if (!fs::is_dir(local_dir))
 	{
-		return CELL_GAME_ERROR_ACCESS_ERROR;
```

Comment:
Better to check gs::g_tls_error for error::exist (is a file) or error::noent (doesn't exist)